### PR TITLE
Add JS Framework : Rexuws

### PR DESCRIPTION
**Pull request by Aderinom**
Hello Community! 

I found a express adapter for uWebsockets.js, Rexuws.
https://github.com/rexuws/rexuws

I want to see how well this does in the benchmarks,to see if I should start putting some work into it. 

With this PR I want to add it to the JS benchmarks.

### CODE REVIEW

File: javascript/rexuws/config.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+framework:
+  website: https://github.com/rexuws/rexuws
```
Comment (waghanza) [inline javascript/rexuws/config.yaml:2]: not useful (website will target github due to next config)

File: javascript/rexuws/config.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  github: rexuws/rexuws
+  version: 1.1.4
```
Comment (waghanza) [inline javascript/rexuws/config.yaml:4]:
```suggestion
  version: 1.1
```

File: javascript/rexuws/package.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+    "dependencies": {
+        "rexuws": "https://github.com/Aderinom/rexuws-update-deps"
```
Comment (waghanza) [inline javascript/rexuws/package.json:3]: a fork ?

**Review by waghanza — CHANGES_REQUESTED** [review]

**Comment by waghanza** [comment]
I only can accept this `PR` after https://github.com/rexuws/rexuws/pull/27 is released @Aderinom 

However, I appreciate your participation, and feel free to contribute for anything else (there is some issue listing javascript frameworks, like `Adonis`)

File: javascript/rexuws/package.json (hunk 3 above)
Comment (Aderinom) [reply javascript/rexuws/package.json:3]: Yes. Updated the version of uws to allow newer node version to run it. 
Didn't really think about the ergononics at that point though.

Already read your comment. Makes sense to block it until then.

**Comment by Aderinom** [comment]
Pushed a possible workaround for the incompability of the older rexuws version.
Added the newer UWS version as a override. Allows the server to run under Node 18.14.

If that's too hacky I understand. 

Also changed the config.yaml as you requested.

File: javascript/rexuws/app.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+app.get("/user/:id", function (req, res) {
+  res.send(req.params.id);
+});
+
+app.post("/user", function (req, res) {
+  res.send("");
+});
+
+app.listen(3000, function () {
+  console.log("listening on port 3000");
```
Comment (waghanza) [inline javascript/rexuws/app.js:31]: I think this could mess-up the results a little bit

File: javascript/rexuws/package.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
+    "scripts": {
+        "run": "node app.js"
+    },
+    "dependencies": {
+        "rexuws": "^1.1.4",
```
Comment (waghanza) [inline javascript/rexuws/package.json:6]:
```suggestion
        "rexuws": "~1.1.4",
```

the idea is to have the same operation type in all package.json for javascript (not sure which is more standard)

**Review by waghanza — CHANGES_REQUESTED** [review]
let's wait a little bit to merge this (maybe in the next round), just to make sure the author accept https://github.com/rexuws/rexuws/pull/27

File: javascript/rexuws/package.json (hunk 5 above)
Comment (Aderinom) [reply javascript/rexuws/package.json:6]: Sure, makes sense to limit it to only minor version changes.

File: javascript/rexuws/app.js (hunk 4 above)
Comment (Aderinom) [reply javascript/rexuws/app.js:31]: This only runs once after the socket has successfully been bound, not on every request.

So this should not cause any difference in req/s.

Comment (Aderinom) [reply javascript/rexuws/app.js:31]: Buuut... since it's not required I guess I'll remove it

**Comment by waghanza** [comment]
I'll close this PR, and rise it again when `rexuws` 2.x will be out

**Comment by Aderinom** [comment]
Yes, sorry. I completely forgot about this!

**Pull request closed without merging**